app/main/authorize/keycloak_manager.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import keycloak
+from flask import current_app, session
+
+
+def decode_token(access_token):
```

Comment:
we can replace lines 50-57 in `app/main/authorize/access_token_sign_in_required.py` with this call now

Reply:
created a new ticket AYR-528 and put the details of changes required

Reply:
I will change in this pr AYR-445

Reply:
code has been updated in access_token_sign_in_required.py to use decode_token function from keycloak_manager